build.rs

```diff
@@ -0,0 +1,17 @@
+#![deny(warnings)]
+
+use std::env;
+
+fn main() {
+    let target = env::var("TARGET").expect("TARGET was not set");
+    if target.contains("uwp") {
```

Comment:
We should check for `-uwp-` so this doesn't accidentally trigger if a target arch/os contains "uwp" as a sub-string.